exporter/otlp/README.md

```diff
@@ -40,7 +40,7 @@ OpenTelemetry::SDK.configure do |c|
   c.add_span_processor(
     OpenTelemetry::SDK::Trace::Export::BatchSpanProcessor.new(
       OpenTelemetry::Exporter::OTLP::Exporter.new(
-        endpoint: 'http://localhost:55680'
+        endpoint: 'http://localhost:4317'
```

Comment:
This is actually not a great example. It can lead to misinterpretation and copy-paste errors. A more reasonable example is to configure compression:
```suggestion
        compression: 'gzip'
```

Reply:
OK. I'll remove the `endpoint` here.  otlp exporter has set the default endpoint with port `4317`.
Thanks.